test/xds/xds_server_integration_test.go

```diff
@@ -222,9 +224,21 @@ func (s) TestServerSideXDS_FileWatcherCerts(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			managementServer, nodeID, bootstrapContents, resolver, cleanup1 := e2e.SetupManagementServer(t, e2e.ManagementServerOptions{})
-			defer cleanup1()
-
+			managementServer := e2e.StartManagementServer(t, e2e.ManagementServerOptions{AllowResourceSubset: true})
```

Comment:
same here: why not use `setupManagementServerAndResolver`

Reply:
Done.